### PR TITLE
[Tests-Only] refactor tags and trashbin tests to not use skeleton files

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Refactor trashbin tests to remove the use of skeleton files

## Related Issue

- Related https://github.com/owncloud/QA/issues/659

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -216,6 +216,10 @@ Given('the following files/folders/resources have been deleted by user {string}'
   return client
 })
 
+Given('user {string} has deleted file {string}', function(user, file) {
+  return webdav.delete(user, file)
+})
+
```
Comment (Talank) [inline tests/acceptance/stepDefinitions/filesContext.js:222]: I didn't find this step used anywhere in this PR. Is it necessary? 🤔

File: tests/acceptance/features/webUITags/createTags.feature [hunk 2]
```diff
@@ -23,6 +24,8 @@ Feature: Creation of tags for the files and folders
 
   @skip @yetToImplement
   Scenario: Create a new tag that does not exist for a file in a folder
+    Given user "Alice" has created a new share with following settings
```
Comment (saw-jan) [inline tests/acceptance/features/webUITags/createTags.feature:27]: Will this step work? Where are the following settings?

File: tests/acceptance/features/webUITags/createTags.feature [hunk 3]
```diff
@@ -32,7 +35,10 @@ Feature: Creation of tags for the files and folders
 
   @skip @yetToImplement
   Scenario: Add a new tag that already exists for a file in a folder
-    Given the user has browsed directly to display the details of file "lorem.txt" in folder "simple-folder"
+    Given user "Alice" has created a new share with following settings
```
Comment (saw-jan) [inline tests/acceptance/features/webUITags/createTags.feature:38]: I think you accidentally added this step? Do we need to create a share here?

File: tests/acceptance/features/webUITags/createTags.feature [hunk 4]
```diff
@@ -43,14 +49,18 @@ Feature: Creation of tags for the files and folders
 
   @skip @yetToImplement
   Scenario: Remove a tag that already exists for a file in a folder
-    Given the user has browsed directly to display the details of file "lorem.txt" in folder "simple-folder"
+    Given user "Alice" has created a new share with following settings
```
Comment (saw-jan) [inline tests/acceptance/features/webUITags/createTags.feature:52]: I think you accidentally added this step? Do we need to create a share here?

File: tests/acceptance/features/webUITags/createTags.feature [hunk 5]
```diff
@@ -43,14 +49,18 @@ Feature: Creation of tags for the files and folders
...
+    And user "Alice" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
+    And the user has browsed directly to display the details of file "lorem.txt" in folder "simple-folder"
     And the user has added a tag "lorem" to the file using the webUI
     When the user browses directly to display the details of file "lorem.txt" in folder "simple-folder"
     And the user toggles a tag "lorem" on the file using the webUI
     Then file "simple-folder/lorem.txt" should have no tags for user "Alice"
 
   @skip @yetToImplement
   Scenario: Create and add tag on a shared file
+    Given user "Alice" has created a new share with following settings
```
Comment (saw-jan) [inline tests/acceptance/features/webUITags/createTags.feature:62]: `following settings` are missing

File: tests/acceptance/features/webUITags/createTags.feature [hunk 6]
```diff
@@ -67,6 +77,8 @@ Feature: Creation of tags for the files and folders
 
   @skip @yetToImplement
   Scenario: Delete a tag in a shared file
+    Given user "Alice" has created a new share with following settings
```
Comment (saw-jan) [inline tests/acceptance/features/webUITags/createTags.feature:80]: `following settings` are missing
Even though the tests are skipped you  can add the data table

File: tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature [hunk 7]
```diff
@@ -5,18 +5,24 @@ Feature: files and folders can be deleted from the trashbin
...
       | name              |
-      | data.zip          |
       | lorem.txt         |
-      | lorem-big.txt     |
       | simple-folder     |
       | sample,1.txt      |
-      | Folder,With,Comma |
+      | lorem-big.txt      |
+      | Folder,With,Comma     |
+      | data.zip     |
```
Comment (saw-jan) [inline tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:24]: some indentation fixes

**Review by swoichha — COMMENTED** [review]
LGTM

File: tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature [hunk 8]
```diff
@@ -5,18 +5,24 @@ Feature: files and folders can be deleted from the trashbin
...
     And the following files have been deleted by user "Alice"
       | name              |
-      | data.zip          |
       | lorem.txt         |
-      | lorem-big.txt     |
       | simple-folder     |
       | sample,1.txt      |
+      | lorem-big.txt     |
       | Folder,With,Comma |
+      | data.zip          |
```
Comment (individual-it) [inline tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:24]: why changing the order?

**Review by individual-it — APPROVED** [review]

File: tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature (hunk 8 above)
Comment (dpakach) [reply tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature:24]: I think the ide did that automagically

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14682/

<p>




</p></details>

**Pull request merged**